src/Internal/Proxy/LegacyProxy.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Class LegacyProxy
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Proxy;
+
+use Exception; // ToDo: Replace this with another exception.
+
+/**
+ * Legacy Proxy
+ *
+ * Used for accessing legacy code, incl. functions, static methods, and globals.
+ * Classes are handled through WCPay\Internal\DependencyManagement\DelegateContainer\LegacyContainer.
+ */
+class LegacyProxy {
+	/**
+	 * Calls a function outside of `src`.
+	 *
+	 * Use this for WP, WC, and other generic non-native PHP functions.
+	 *
+	 * @param string $name          Name of the function.
+	 * @param mixed  ...$parameters Parameters to pass to the function.
+	 *
+	 * @return mixed The response from the function.
+	 */
+	public function call_function( string $name, ...$parameters ) {
+		return call_user_func_array( $name, $parameters );
```

Comment:
My thinking is that trying to access an undefined global variable would only generate a notice, but functions and methods kind of should just be there when we're trying to access them, throwing a fatal error otherwise.

With that in mind, what are your thoughts on having a separate `is_callable()` method within the proxy?

I know this is a small detail, but we use many functions to connect with the outside world, and I don't want to make it seem like each of them requires a `try/catch`. WDYT? I just wanted to discuss it a bit first, but I'm fine either way.